src/pages/Admin/Home/index.tsx

```diff
@@ -280,6 +280,8 @@ type AdminVoucherModalProps = Omit<ModalProps, "children" | "isOpen"> & {
   type?: types | null
 }
 
+var uploadDisabled = false;
```

Comment:
Can you change this to using a useState hook, something like:

```
const [isUploadDisabled, setIsUploadDisabled] = useState(false)
```

Then consecutively if you want to change it, can do it by: `setIsUploadDisabled(true)`